README.md

```diff
@@ -1,2 +1,68 @@
 # node-api
 ## An ultra-lightwight API framework using Express.js
+
+### Pre-requsites
+
+##### [Postgres](http://www.postgresql.org/)
+
+This project uses a Postgres database by default. You can install Postgres any way you want. On OSX with Homebrew you can:
+
+```
+brew install postgresql
+```
+
+You'll need to create two databases if you're developing locally, one for the application to use and the other for the test suite. You can choose the names of these databases (typically `<app name>` and `<app name>-test`) by setting the environment variables `DATABASE_URL` and `TEST_DATABASE_URL` (see [direnv](#direnv) below). Once you've done that, you can create the databases using the `createdb` command,
```

Comment:
I reckon expanding this step is needed, i'm a newb to Postgres so a link to an intro article would be helpfull.